pages/03.Tutorials/11.manage-worker-nodes-via-cli/default.en.md

```diff
@@ -19,80 +19,89 @@ To get a list of all nodes execute:
+A machinedeployment looks like the following:
 
-cat <<EOF | kubectl apply -f -
+```yaml
 apiVersion: cluster.k8s.io/v1alpha1
-kind: Machine
+kind: MachineDeployment
 metadata:
-  name: machine-kubermatic-${CLUSTER_NAME}-${MACHINE_NAME}
+  annotations:
+    machinedeployment.clusters.k8s.io/revision: "2"
+  creationTimestamp: "2019-02-18T18:10:57Z"
+  generateName: kubermatic-92r62cx8j6-
+  generation: 2
+  name: kubermatic-92r62cx8j6-xmljg
   namespace: kube-system
+  resourceVersion: "4015"
+  selfLink: /apis/cluster.k8s.io/v1alpha1/namespaces/kube-system/machinedeployments/kubermatic-92r62cx8j6-xmljg
+  uid: 89e8b9f0-33a8-11e9-9e48-0a580af40383
 spec:
-  metadata:
-    name: kubermatic-${CLUSTER_NAME}-${MACHINE_NAME}
-  providerSpec:
-    value:
-        cloudProvider: openstack
-        cloudProviderSpec:
-          availabilityZone: ${AVAILABILITY_ZONE}
-          flavor: ${FLAVOR}
-          floatingIpPool: ${FLOATING_IP_POOL}
-          identityEndpoint: "https://api.${REGION}.cloud.syseleven.net:5000/v3"
-          image: "${IMAGE_NAME}"
-          network: kubermatic-${CLUSTER_NAME}
-          region: ${REGION}
-          securityGroups:
-          - kubermatic-${CLUSTER_NAME}
-        operatingSystem: ${OPERATING_SYSTEM}
-        operatingSystemSpec:
-          distUpgradeOnBoot: false
-        sshPublicKeys:
-        - "${SSH_PUBLIC_KEY}"
-  roles:
-  - Node
-  versions:
-    kubelet: "v${K8S_VERSION}"
-EOF
+  minReadySeconds: 0
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 1
+  selector:
+    matchLabels:
+      machine: md-92r62cx8j6-cc45b75qqf
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        machine: md-92r62cx8j6-cc45b75qqf
+    spec:
+      metadata:
+        creationTimestamp: null
+      providerSpec:
+        value:
+          cloudProvider: openstack
+          cloudProviderSpec:
+            availabilityZone: dbl1
+            domainName: ""
+            flavor: m1.micro
+            floatingIpPool: ext-net
+            identityEndpoint: https://api.dbl.cloud.syseleven.net:5000/v3
+            image: Ubuntu 18.04 LTS sys11 optimized - 2018-08-13
+            network: kubermatic-92r62cx8j6
+            password: ""
+            region: dbl
+            securityGroups:
+            - kubermatic-92r62cx8j6
+            subnet: 86db31c3-d5dd-4a88-bc92-ab8a148062c8
+            tags:
+              kubermatic-cluster: 92r62cx8j6
+            tenantName: ""
+            tokenId: ""
+            username: ""
+          operatingSystem: ubuntu
+          operatingSystemSpec:
+            distUpgradeOnBoot: false
+          sshPublicKeys: []
+      versions:
+        kubelet: 1.13.3
 ```
 
-For more details on Machines see [Cluster Management API](../../02.Documentation/12.cluster-api/default.en.md).
+The most common fields to update are:
```

Comment:
There should be some note here, that the machine-controller is doing a rolling update of the deployment and how it does it (cordon, drain, recreate)